app/component/map/map.cjsx

```diff
@@ -27,36 +27,20 @@ class Map extends React.Component
 
   @contextTypes:
     getStore: React.PropTypes.func.isRequired
-
-  @currentLocationIcon:
-    if isBrowser
-      L.divIcon
-        html: Icon.asString 'icon-icon_mapMarker-location-animated'
-        className: 'current-location-marker'
-    else
-      null
-
-  getLocation: ->
-    coordinates = @context.getStore('PositionStore').getLocationState()
-    if coordinates and (coordinates.lat != 0 || coordinates.lon != 0)
-      coordinates: [coordinates.lat, coordinates.lon]
-      zoom: 16
-      hasPosition: true
-    else
-      coordinates: [60.17332, 24.94102]
-      zoom: 11
-      hasPosition: false
+    executeAction: React.PropTypes.func.isRequired
 
   setBounds: (props) ->
     @refs.map.getLeafletElement().fitBounds(
       [props.from, props.to],
       paddingTopLeft: props.padding)
 
-  componentDidMount: ->
-    @context.getStore('PositionStore').addChangeListener @onChange
+  componentDidMount: =>
     @context.getStore('EndpointStore').addChangeListener @onChange
     L.control.attribution(position: 'bottomleft', prefix: false).addTo @refs.map.getLeafletElement()
-    if not (@props.disableZoom or L.Browser.touch)
+    if @props.disableMapTracking
+      @refs.map.getLeafletElement().addEventListener('dragstart', @props.disableMapTracking)
+      @refs.map.getLeafletElement().addEventListener('zoomend', @props.disableMapTracking)
```

Comment:
These can be added as props (onLeafletDragstart and onLeafletZoomend) on the map object